app/javascript/article-form/components/__tests__/ArticleCoverImage.test.jsx

```diff
@@ -46,14 +46,14 @@ describe('<ArticleCoverImage />', () => {
     });
 
     it('shows the change and remove buttons', () => {
-      const { getByText } = render(
+      const { queryByText } = render(
         <ArticleCoverImage
           mainImage={'/some-fake-image.jpg'}
           onMainImageUrlChange={jest.fn()}
         />,
       );
-      getByText('Change');
-      getByText('Remove');
+      expect(queryByText('Change')).toBeDefined();
```

Comment:
As mentioned in a previous PR, I opted for queryBy as it makes the tests more readable by using an explicit `expect` to test for existence. See the [Testing Library cheatsheet](https://testing-library.com/docs/react-testing-library/cheatsheet) for the difference between the two.